streaming/util/importing.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utilities for importing."""
+
+import inspect
+import sys
+from importlib import import_module
+from warnings import warn
+
+__all__ = ['get_import_exception_message', 'redirect_imports']
+
+
+def get_import_exception_message(package_name: str, extra_deps: str) -> str:
+    """Get import exception message.
+
+    Args:
+        package_name (str): Package name.
+
+    Returns:
+        str: Exception message.
+    """
+    return f'Streaming was installed without {package_name} support. ' + \
+            f'To use {package_name} related packages with Streaming, run ' + \
+            f'`pip install \'mosaicml-streaming[{package_name}]\'`.'
+
+
+def redirect_imports(new_fqdn: str) -> None:
```

Comment:
has this method been tested? just want to make sure that the indexing into `inspect.stack()` and so on to get the current module is working as intended

Reply:
added test